content/docs/configurations.html

```diff
@@ -524,7 +524,7 @@ <h4 id="HIVE_USE_JDBC_OPT_KEY">HIVE_USE_JDBC_OPT_KEY</h4>
 
 <h4 id="HIVE_AUTO_CREATE_DATABASE_OPT_KEY">HIVE_AUTO_CREATE_DATABASE_OPT_KEY</h4>
 <p>Property: <code class="highlighter-rouge">hoodie.datasource.hive_sync.auto_create_database</code> Default: <code class="highlighter-rouge">true</code> <br />
-<span style="color:grey"> Auto create hive database if does not exists </span></p>
+<span style="color:grey"> Auto create hive database if does not exists. <b>Note</b>: for versions 0.7 and 0.8 you will have to explicitly set this to true </span></p>
```

Comment:
@veenaypatil you should update the .md file instead of html file.

Reply:
@leesf  oh ok, updated the .md file now, pls check